### PR TITLE
Fix Indowebnovel 

### PR DESCRIPTION
**Pull request by AhmadNaruto**
i don't know why but advance-search is temporary not function in website.
so until it fixed by website admin, i use normal search

### CODE REVIEW

File: plugins/indonesian/indowebnovel.ts [hunk 1]
```diff
@@ -112,14 +114,16 @@ class IndoWebNovel implements Plugin.PluginBase {
...
   ): Promise<Plugin.NovelItem[]> {
+    /*
     let link = `${this.site}advanced-search/page/${page}/?title=${searchTerm}&author=&yearx=`;
     link += `&status=${this.filters.status.value}`;
     link += `&type=${this.filters.type.value}`;
     link += `&order=${this.filters.sort.value}`;
     link += this.filters.lang.value.map(i => `&country[]=${i}`).join('');
-
+    */
+    let link = '{this.site}page/${page}/?s=${searchTerm}'
```
Comment (nyagami) [inline plugins/indonesian/indowebnovel.ts:126]: this is invalid url

Comment (AhmadNaruto) [reply plugins/indonesian/indowebnovel.ts:126]: i have fix it 👍

**Comment by nyagami** [comment]
Skete Novel icon is missing, pls add it

**Review by nyagami — APPROVED** [review]

**Comment by AhmadNaruto** [comment]
> Skete Novel icon is missing, pls add it

done 👍

**Pull request merged**